.pre-commit-config.yaml

```diff
@@ -11,7 +11,7 @@ repos:
         exclude: examples/WTMetaD_paper/r2/free_energy/wtmetad_ib/accumulated_bias/bias_after_iter_15.dat
       - id: check-added-large-files
         args: ['--maxkb=500', '--enforce-all']
-        exclude: tests/data/data.zip
+        exclude: (?x)^(tests/data/data.zip | tests/data/water_al.npz)$
```

Comment:
I think we could still try to make this file smaller, i.e. by reading it and saving using the old mlptrain version and making sure we use old numpy as well? Worth a try.